react/components/Alert/README.md

```diff
@@ -14,47 +14,59 @@
 
 For low-priority notification prefer a <a href="#/Components/Notification/ToastProvider">Toast</a>.
 
+## Examples
```

Comment:
We do not use such title in the other components docs. I'd use a space or maybe `hr` to separate examples from explanations above.

Reply:
Got it! Changed it in deb0abc

Reply:
Nice!